apollo-federation/src/connectors/json_selection/methods/public/gte.rs

```diff
@@ -0,0 +1,300 @@
+use apollo_compiler::collections::IndexMap;
+use serde_json_bytes::Value as JSON;
+use shape::Shape;
+use shape::location::SourceId;
+
+use crate::connectors::json_selection::ApplyToError;
+use crate::connectors::json_selection::ApplyToInternal;
+use crate::connectors::json_selection::MethodArgs;
+use crate::connectors::json_selection::VarsWithPathsMap;
+use crate::connectors::json_selection::immutable::InputPath;
+use crate::connectors::json_selection::location::Ranged;
+use crate::connectors::json_selection::location::WithRange;
+use crate::impl_arrow_method;
+
+impl_arrow_method!(GteMethod, gte_method, gte_shape);
+/// Returns true if the applied to value is greater than or equal to the argument value.
+/// Simple examples:
+///
+/// $->echo(3)->gte(3)       results in true
+/// $->echo(4)->gte(3)       results in true
+/// $->echo(2)->gte(3)       results in false
+/// $->echo("a")->gte("b")   results in false
+/// $->echo("c")->gte("b")   results in true
+fn gte_method(
+    method_name: &WithRange<String>,
+    method_args: Option<&MethodArgs>,
+    data: &JSON,
+    vars: &VarsWithPathsMap,
+    input_path: &InputPath<JSON>,
+) -> (Option<JSON>, Vec<ApplyToError>) {
+    if let Some(MethodArgs { args, .. }) = method_args {
+        if let [arg] = args.as_slice() {
+            let (value_opt, arg_errors) = arg.apply_to_path(data, vars, input_path);
```

Comment:
```suggestion
    if let Some(first_arg) = method_args.and_then(|args| args.args.first()) {
        let (value_opt, arg_errors) = first_arg.apply_to_path(data, vars, input_path);
```
We do this style elsewhere, and I like it.